### DISCUSSION TITLE
Which version of MXNet to install with CUDA 11.3

### DISCUSSION THREAD

**Discussion by tripleaamin** [apache/incubator-mxnet#20477] [Q&A]
Hi, I was wondering what version of mxnet I should be installing if I am working with CUDA 11.3.

The latest version I have found is mxnet-cu112. Which doesn't seem to be compatiable.

**Comment by leezu** [comment]
If mxnet-cu112 doesn't work, you can compile from source: https://mxnet.apache.org/versions/1.8.0/get_started/build_from_source
